.travis.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - stage: build
       if: type IN (push, pull_request)
       install: ./gradlew assemble
-      script: ./gradlew check
+      script: ./gradlew -x :bindings:check -x :standalone:check check
```

Comment:
It just occurred to me that this may not be what we want. If we don't run `check` on `bindings` or `standalone` then we could end up with code degradation or even uncompilable code.

Are these built and checkstyled at least? We may not want the tests to run, but it should still be compiled.

Reply:
I think the intuition is right that we don't need to run the tests for bindings upon every single PR build, specially if there is no change in the bindings. The interface abstraction between the segment store and tier 2 should be sufficient to validate changes elsewhere without running all tier 2 tests.

What's bad about this line change:
- If we make changes to tier 2 bindings, then we won't be running tier 2 tests
- We also need a consistent way to test standalone, and I believe changes across pravega can affect the behavior of standalone. We could have a daily standalone build, but we would still have the same issue that changes focused on standalone (which happen rarely) would not trigger standalone tests with this definition.

Reply:
These changes are limited to travis. All the other places where we run periodic (daily) tests will not be affected by this change.